modules/scheduler-impl/src/main/java/org/opencastproject/scheduler/impl/SchedulerServiceImpl.java

```diff
@@ -1523,6 +1523,7 @@ private void removeSchedulingInfoFromIndex(String mediaPackageId) {
 
     try {
       index.addOrUpdateEvent(mediaPackageId, updateFunction, orgId, user);
+      index.deleteEvent(mediaPackageId, orgId);
```

Comment:
Is only deleting the event not sufficient? Why does it have to be updated and deleted afterward?

Reply:
In theory it is. I really don't know why there was a step specifically keeping the event with an empty capture agent ID.

Reply:
As far as I can tell, updating the event before deletion has no beneficial side-effects. Therefore I went ahead and removed the update related code. Do you agree with this change @geichelberger ?